# Task Id
uint64 task_id

# Task Type
uint32 task_type

# Target Position
geometry_msgs/Pose target_position

# Target Object
string target_object

# Task State
uint8 task_state

# Task Result
uint8 task_result

# 到达终点是否原地打转
uint8 isInPlaceRotation 
